packages/rum-core/src/domain/contexts/ciTestContext.ts

```diff
@@ -5,8 +5,13 @@ export interface CiTestWindow extends Window {
 }
```

Comment:
I think you can simply add this to the interface
```suggestion
  dd_ci_visibility_test_context?: {
    test_execution_id: string
  }
}
```